modules/chat/src/main/ChatApi.scala

```diff
@@ -63,7 +67,8 @@ final class ChatApi(
           .ifFalse(mine.isEmpty)
           .so:
             chatTimeout.isActive(chat.id, _)
-      yield UserChat.Mine(mine, lines, timeout)
+        blockedUsers <- me.so(relationApi.fetchBlocking(_))
+      yield UserChat.Mine(mine, lines, timeout, false, Some(blockedUsers))
```

Comment:
This fetches up to 500 blocked user IDs from the database and sends them with every page a user loads.

It's not acceptable

Reply:
Updated to only send blocked IDs of those who have already written in the chat. Only drawback is if the user is already in the chat before a blocked person types their first message, the user will have to refresh once to not see their messages going forward.